tools/fab/frontend-install.py

```diff
@@ -61,18 +61,13 @@ def banner(message):
 	print('#######################################')
 
 
-def copy(source, dest):
-	isodir = tempfile.tempdir()
-	banner("Copying %s to local disk" % source)
-	mount(source, isodir)
-	subprocess.call(['mkdir', '-p', dest])
-	subprocess.call(['cp', '-r', isodir, dest])
-	umount(isodir)
-
-
 def mount(source, dest):
 	subprocess.call(['mkdir', '-p', dest])
-	subprocess.call(['mount', '-o', 'loop,ro', source, dest])
+	proc = subprocess.Popen(['mount', '-o', 'loop,ro', source, dest], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+	result = proc.communicate()
+	if proc.returncode != 0:
+		proc = subprocess.Popen('mount | grep iso', shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
Is this piping sdterr into stdout on purpose? Or was that an artifact of debugging?

Reply:
Purpose was to get unified output (stdout and stderr printed interwoven as they would appear on the console).  Same as `command >out 2>&1` in `bash`.

Along those lines, the print is intentional (it will only happen if the script is likely to fail), although it should also print the error message of the original mount attempt (useful for debugging).  I'll add that.